### ISSUE TITLE
swap deployment:specific container --run-shell

### ISSUE THREAD

**Issue by lucidprogrammer** [telepresenceio/telepresence#2851]
I am using telepresence version as follows. It is not the latest v2 version, however it seems thats the one which seems to work with EKS v1.22. 
```
Client: v2.6.6 (api v3)
Root Daemon: v2.6.6 (api v3)
User Daemon: v2.6.6 (api v3)

```
In the legacy telepresence, I used to swap a deployment like the following,

```
telepresence --namespace my-namespace --swap-deployment my-operator-deployment:manager --run-shell
```
In the above scenario, my-operator-deployment is a deployment of a golang operator, and manager is a container within that, so I swap and do local development by running the code directly on my machine. (Service/port is immaterial in this use case)

How do I get this functionality in V2, or is it possible? Following is what I did
1. installed telepresence cli
2. installed telepresence in the cluster using helm
3. telepresence connect
then the following command,

```
telepresence --namespace my-namespace intercept my-operator-deployment:manager -- bash
telepresence: error: workload "my-operator-deployment:manager.my-namespace" not found
```
In the above, if I give a --service metrics, it works, but that does not give the facility which I need. 

So how do I get a similar functionality as the legacy command, in the v2 version, where I swap the deployment selecting a sepecific container as in `telepresence --namespace my-namespace --swap-deployment my-operator-deployment:manager --run-shell`

**Comment by cindymullins-dw** [comment]
HI @lucidprogrammer , I don’t believe there’s an equivalent to that currently.  You can specify the Service but not the container per se.  If there are multiple containers, Telepresence will either try to choose one (auto-detecting which one to use) or may prompt you to specify the Service if there’s multiple Service - what do you observe there? The `workload not found error `is likely because you're passing in the `:manager` which is incorrect syntax. Could you try passing in Service.manager and see if that works?

**Comment by cindymullins-dw** [comment]
Hi @lucidprogrammer, for users who liked the v1 Telepresence functionality of 'swap-deployment' we have a [`--replace` flag](https://www.getambassador.io/docs/telepresence/latest/reference/intercepts/cli#replacing-a-running-workload) that is now available in 2.16.1.

**Comment by lucidprogrammer** [comment]
1. Yes I see the --replace flag working in 2.16.1. However, in subsequent releases, it is not there.
```
$ telepresence version
OSS Client : v2.17.0
Root Daemon: not running
User Daemon: not running
$ telepresence intercept my-service --port 8080 --replace
telepresence intercept my-service --port 8080 --replace
telepresence intercept: error: unknown flag: --replace
```
2. I then downloaded the specific version you specified above by @cindymullins-dw , I do get the replace flag. However, still not able to use for my use case. I don't have a service to replace, I just want to replace one running container, it is a standard kubernetes operator process, which does reconciliation on detecting a custom resource. Possibly what is needed is a way to specify container name.
